.github/workflows/ci.yml

```diff
@@ -56,60 +94,131 @@ jobs:
       - name: Lint
         run: just lint
 
-  kt-test:
+  kotlin-build-test-deploy-snapshot:
+    needs:
+      - build_aarch64_apple_darwin
+      - build_x86_64_apple_darwin
+      - build_x86_64_unknown_linux_gnu
+      - build_x86_64_unknown_linux_musl
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: true
-      - name: Init Hermit
-        uses: cashapp/activate-hermit@v1
+      - name: Set up JDK 11
```

Comment:
no more hermit? we just hard committing to jdk 11 because its what cash uses?

Reply:
Yes @nitro-neal - details alongside what @KendallWeihe said: the Hermit config is bringing in an x86_64 JDK on aarch64 environments, and that leads to running the JVM in virtualized mode and picking up the wrong native binary...so, we go manual in this CI for now to keep things working